src/taskgraph/run-task/run-task

```diff
@@ -1053,9 +1054,14 @@ def maybe_run_resource_monitoring():
     monitor_process.start()
     return process
 
+def _display_pyhton_version():
+    print("Python version: ")
+    print(platform.python_version())
+
 
 def main(args):
     os.environ["TASK_WORKDIR"] = os.getcwd()
+    _display_pyhton_version()
```

Comment:
Let's move this after the `print_line` call below. That log line signals the very start of the script, so it makes sense to me that it would be the first line in the log.

Reply:
ok, I will update that

Reply:
Oups, yes, thanks, I have fixed it.